### DISCUSSION TITLE
Just wanted to say, the new 3D Terrain feature is freaking awesome!!

### DISCUSSION THREAD

**Discussion by ahmedkhelifi** [maplibre/maplibre-native#555] [Ideas]
I've spent the last few weeks playing around with the Maplibre-gl new 3D support for my desktop App and Mapbox's new ios SDK and oh man the 3D support really awesome!

I'm planning few hiking/climbing trips in the alps and being able to see the mountains in 3D just gives you a whole new perspective with how much you can plan. Using it during a live hike you can better visualize when the climb is finally going to end and when you will be able to catch your breath again 😅

I've seen the topic come up here and there and bit, but don't know if this is on the team's roadmap. In any case, I hope this posts  encourages the team to keep it or put it in the roadmap to bring it to native devices as well 😊

**Comment by wipfli** [comment]
Thanks for the feedback @ahmedkhelifi. I also like Terrain3D for mountain stuff...
